### DISCUSSION TITLE
Can we have a `size` prop on <Menu /> component ?

### DISCUSSION THREAD

**Discussion by thevipinmishra** [mantinedev/mantine#1948] [Feature requests]
I would love to have different sizes in `<Menu />` component, I can't see it in the docs or any of the examples.

Can we have a `size` prop on `<Menu />` component please ?

**Comment by Paraxdev** [comment]
so, kinda like https://mantine.dev/core/menu/#custom-component-as-target does?

**Reply by thevipinmishra** [reply]
Ofcourse we can have our custom implementation using `createStyles` api or custom menus using Popover etc. but having a prop is straight forward most of the times. 🙂

**Comment by rtivital** [comment]
You can set Menu width in px: `<Menu width={320} />`

**Reply by thevipinmishra** [reply]
I'm sorry if I didn't explain it will.

By size, I mean the the size of the Menu in terms of font-size and padding.

Similar to what we do with with `<Button size="xl" />` to create a larger/bigger button.
